test/karma.conf.js

```diff
@@ -14,7 +14,7 @@ const browserStack = {
   accessKey: process.env.BROWSERSTACK_ACCESS_KEY,
   build,
   // https://github.com/browserstack/api#timeout300
-  timeout: 2.5 * 60, // Maximum time before a worker is terminated. Default 5 minutes.
+  timeout: 5 * 60, // Maximum time before a worker is terminated. Default 5 minutes.
```

Comment:
For context, the higher the value is, the more we potentially slow down the CI when CircleCI timeout (he can no longer cancel the build on BrowserStack.
```suggestion
  timeout: 5 * 60, // Maximum time before a worker is terminated. Default 5 minutes.
```

Reply:
But it still getting to timeout :/

Reply:
![Screenshot_2021-07-05-17-11-54-878_com brave browser](https://user-images.githubusercontent.com/3165635/124491988-2afce280-ddb4-11eb-9e90-de45b7f50528.jpg)

Maybe, but the increase of timeout worked. Waited 5 minutes, but not BrowserStack runner was available, before CircleCI timeouted.

Reply:
It passed the 2nd time : v

Reply:
And it wasn't this slow
![Screenshot_2021-07-05-17-36-10-111_com brave browser](https://user-images.githubusercontent.com/3165635/124494823-9300f800-ddb7-11eb-9c1a-bedcb5b58cfb.jpg)

Reply:
Maybe we can try 3mn30 or 4mn to avoid slowing the CI to much

Reply:
4 minutes sounds great. Plus, it won't matter so much once we stop our current "fire & forget" strategy, triggering builds in BrowserStack that we know will complete after the 30minutes timeout delay of CircleCI for builds.

Reply:
If we have a way to fail the step before the 30mn timeout, that would be a great improvement.
I'm setting it to 4mn then.

Reply:
Yeah, needs to rewrite the Karma plugin for this, it's not really hard, but could take a day, we avoided it so far 😁. Or we kill the browser stack tests, or we pay, or we migrate to a Docker selenium grid.

Reply:
OK, I suppose it's worth seeing how it behaves with this PR